app/models/user.rb

```diff
@@ -16,6 +16,7 @@ class User < ActiveRecord::Base
     :pto_hours_for_date_range, to: :days
 
   scope :active, -> { where(active: true) }
+  scope :time_zones, -> { pluck('DISTINCT time_zone') }
```

Comment:
Heh, it was originally a class method, then I decided to get all fancy and scopeify it. I'll change it back.